tests/test_dataset.py

```diff
@@ -69,6 +69,8 @@ def test_torchvision_dataset():
     assert isinstance(data.dataset[0][0], str), "ImageFolder expected to return input "\
         "of type 'str', got %s" % type(data.dataset[0][0])
 
+def test_darknet_dataset():
+    assert "DarknetDataset" in dataset.all_datasets['external']
```

Comment:
would be better if you put a proper tests here :)

Reply:
ok

Reply:
more tests added